rkt/stop.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package main
+
+import (
+	"github.com/coreos/rkt/stage0"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/spf13/cobra"
+)
+
+var (
+	cmdStop = &cobra.Command{
+		Use:   "stop UUID ...",
+		Short: "Stop a pod",
+		Run:   runWrapper(runStop),
+	}
+	flagForce bool
+)
+
+func init() {
+	cmdRkt.AddCommand(cmdStop)
+	cmdStop.Flags().BoolVar(&flagForce, "force", false, "forced stopping")
+}
+
+func runStop(cmd *cobra.Command, args []string) (exit int) {
+	var podUUID *types.UUID
+	var podUUIDs []*types.UUID
+	var ret int
```

Comment:
can it be called `errors` for consistency with `rmImages`? Then use `errors++` in case of errors.